implementations/rust/ockam/ockam_api/tests/common/common.rs

```diff
@@ -129,7 +131,7 @@ pub async fn start_authority(
     configuration.no_direct_authentication = false;
     configuration.no_token_enrollment = false;
 
-    println!(
+    debug!(
```

Comment:
I wonder why it was like that here. @polvorin is that intentional?

Reply:
my bad, a leftover. Some test were failing and I wondered why.   Letting it as debug is fine,  or remove it entirely as it doesn't look useful anymore now.